dom.bs

```diff
@@ -5231,17 +5260,17 @@ to return a new {{Comment}} <a for=/>node</a> whose <a for=CharacterData>data</a
 and <a for=Node>node document</a> is <a>this</a>.
 
 <p class=note>No check is performed that <var>data</var> consists of
-characters that match the <code><a type>Char</a></code> production
+characters that match the <code><a>Char</a></code> production
 or that it contains two adjacent hyphens or ends with a hyphen.
 
 <p>The
 <dfn method for=Document><code>createProcessingInstruction(<var>target</var>, <var>data</var>)</code></dfn>
 method steps are:
 
 <ol>
- <li>If <var>target</var> does not match the
+ <li>If <var>target</var> is does not match the
  <!--<code data-anolis-type>PITarget</code>-->
- <code><a type>Name</a></code> production,
+ <code>[=XML/Name=]</code> production,
  then <a>throw</a> an "{{InvalidCharacterError!!exception}}" {{DOMException}}. <!-- DOM3 does not check for "xml" -->
```

Comment:
The added "is" here seems incorrect. Might also want to drop the dated comments while here.